backend/config/urls.py

```diff
@@ -29,6 +29,12 @@
     path('', views.index),
     path('example/', views.example),
     path('example/<example_id>', views.example),
+    path('api/create_person', views.create_person),
+    path('api/create_event', views.create_event),
+    path('api/get_people', views.get_people),
+    path('api/get_people_from_event/<event_name>', views.get_people_from_event),
```

Comment:
We should talk about changing this to take `event_id`s, rather than event names. The reason is: event names might be complicated and long, might be referred to as something in one place, and a substring of that name in another. It's just a lot harder to be exact with a string of names. 

Here is a common pattern:
"api/get_events" or "api/events" yields a list of events with their IDs, names, dates, etc.

"api/get_event/<event_id>" or "api/events/<event_id>" will get you that exact event. So someone using the API might query all events, find their exact event using the list, then have the ID on hand to do all GETs and POSTs with. 

You can also add query params to your list of events:

"api/get_events/?event_name=some_event_string_or_substring". Where by default, we ask folks to use the event ID, but to be nice, we also allow them to query using other parameters like name and date.